### PR TITLE
Split standard-specific parts of address-controller into a new component 'standard-controller'

### PR DESCRIPTION
**Pull request by lulf**
The standard-controller is responsible for managing brokers in the
standard address space and update status of addresses based on broker
and router status.

### CODE REVIEW

**Comment by lulf** [comment]
@grs This is the first phase of moving the controller loop for standard address space into the address space.

**Comment by lulf** [comment]
@grs WDYT about merging this? This only affects the standard address space

File: templates/include/standard-space-infra.jsonnet [hunk 1]
```diff
@@ -32,19 +35,34 @@ local prometheus = import "prometheus.jsonnet";
...
+    ],
+
+    local storage_template_configmap = [
+      templateConfig.storage("enmasse-storage-templates")
+    ],
+
+    local template_config = (if use_template_configmap then "enmasse-storage-templates" else ""),
+
+    "objects": [
+      standardController.deployment(template_config, "authservice-ca"),
```
Comment (grs) [inline templates/include/standard-space-infra.jsonnet]: Is there a reason you prefer this outside the admin pod?

Comment (lulf) [reply templates/include/standard-space-infra.jsonnet]: No, I can move it in there instead. It was a bit easier to experiment with it separately.

Comment (grs) [reply templates/include/standard-space-infra.jsonnet]: I have no objection to merging

**Pull request merged**